packages/server/tsconfig.build.json

```diff
@@ -8,7 +8,7 @@
     "esModuleInterop": true,
```

Comment:
These tsconfig.json are not really related to the issue, I just tidied them up while analysing the issue